deploy/olm-catalog/3scale-community-operator/3scale-community-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -107,29 +107,29 @@ spec:
     Documentation can be found on our [website](https://github.com/3scale/3scale-operator/blob/v0.3.0/doc/user-guide.md).
 
     ### Getting help
-    If you encounter any issues while using 3scale operator, you can create an issue on our [website](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
+    If you encounter any issues while using 3scale community operator, you can create an issue on our [website](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
 
     ### Contributing
     You can contribute by:
 
-    * Raising any issues you find using 3scale Operator
+    * Raising any issues you find using 3scale community Operator
     * Fixing issues by opening [Pull Requests](https://github.com/3scale/3scale-operator/pulls)
     * Improving [documentation](https://github.com/3scale/3scale-operator/blob/v0.3.0/doc/user-guide.md)
-    * Talking about 3scale Operator
+    * Talking about 3scale community Operator
 
     All bugs, tasks or enhancements are tracked as [GitHub issues](https://github.com/3scale/3scale-operator/issues).
 
     ### License
-    3scale Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
+    3scale community Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
 
   keywords:
     - 3scale
     - API
-  displayName: 3scale Operator
+  displayName: 3scale
```

Comment:
3scale community here?

Reply:
See discussion [here](https://github.com/3scale/3scale-operator/pull/212#discussion_r317090653)

What do you think? I think leaving 3scale only makes more sense.

Reply:
I’d say it should also say community here so you know which one you’re using. Why leaving 3scale only makes more sense?

Reply:
from previous [discussion](https://github.com/3scale/3scale-operator/pull/212#pullrequestreview-278945408):

`Community will be removed as well because it is explicit from provider type. RH's Operatorhub displays community watermark when provider type is community`